.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-2.1.5
```

Comment:
Ruby's versioning is pretty meaningless (in my opinion).  It's either 1.8.7 or 1.9.2/3 or 2.X  The minor tweaks between major releases are not going to introduce anything you really care about.  I only upgrade when there are security issues that *must* be addressed.